plugins/elantp/README.md

```diff
@@ -0,0 +1,47 @@
+Elan TouchPad
+=============
+
+Introduction
+------------
+
+This plugin allows updating Touchpad devices from Elan. Devices are enumerated
```

Comment:
Does this also happen to work with touchscreens?  It would be good to confirm if it's the same protocol as the protocol might be renamed and plugin renamed.

Reply:
> Does this also happen to work with touchscreens

I'm lead to believe these are only touchpads, i.e. palm-detection in the keyboard kinda things.